pom.xml

```diff
@@ -42,7 +41,7 @@
     <dependency>
       <groupId>org.apache.struts</groupId>
       <artifactId>struts2-core</artifactId>
-      <version>2.5.14</version>
+      <version>2.5.15</version>
```

Comment:
### org.apache.struts : struts2-core : 2.5.15
⚠️ **No recommended versions** are available for this component (as of _Feb 25, 2020_)

Threat (of 10) | IQ Policy Name | Violation Details
-- | -- | -- | 
9 | Security-High | **High Risk CVSS Score**. Found security vulnerabilities: [CVE-2016-1000338](http://localhost:8070/assets/index.html#/vulnerabilities/CVE-2016-1000344), [CVE-2016-1000340](http://localhost:8070/assets/index.html#/vulnerabilities/CVE-2016-1000344), [CVE-2016-1000342](url), [CVE-2016-1000343](http://localhost:8070/assets/index.html#/vulnerabilities/CVE-2016-1000344), [CVE-2016-1000344](http://localhost:8070/assets/index.html#/vulnerabilities/CVE-2016-1000344)
3 | Security-Low | **Low Risk CVSS Score**. Found security vulnerability: [CVE-2016-6644](http://localhost:8070/assets/index.html#/vulnerabilities/CVE-2016-1000344).